webapp/src/shared/components/Avatar/Avatar.tsx

```diff
@@ -17,7 +17,8 @@ type LargeAvatarProps = AvatarProps & {
   editUrl?: string;
 };
 
-const SMALL_AVATAR_DOWNSCALE = 0.2;
+const SMALL_AVATAR_DOWNSCALE = 0.05;
```

Comment:
I would like to double check this constant, but not now. It appears good, so I'l take it as such.

Reply:
Yes, do it, please. I've just extrapolate from the ones you put